### ISSUE TITLE
ModuleNotFoundError: No module named 'codetf.common'

### ISSUE THREAD

**Issue by souvikqb** [salesforce/CodeTF#33]
ModuleNotFoundError                       Traceback (most recent call last)
[<ipython-input-6-3de5b344913a>](https://localhost:8080/#) in <cell line: 1>()
----> 1 import codetf
      2 print(codetf.__version__)

[/usr/local/lib/python3.10/dist-packages/codetf/__init__.py](https://localhost:8080/#) in <module>
      5 # sys.path.append("../")
      6 from omegaconf import OmegaConf
----> 7 from codetf.common.registry import registry
      8 
      9 root_dir = os.path.dirname(os.path.abspath(__file__))

ModuleNotFoundError: No module named 'codetf.common'

**Comment by Paul-B98** [comment]
@souvikqb are u using the pip package or are u installing it directly from github?

**Comment by souvikqb** [comment]
Hi @Paul-B98 I tried both the methods (Pip & Git) on Colab as well as on a Virtual Machine

**Comment by Paul-B98** [comment]
@souvikqb I tried it with the alternative method (install via github) from the Readme and had no issues. Have u done the following steps or anything else?
```bash
git clone https://github.com/salesforce/CodeTF.git
cd CodeTF
pip install -e .
pip install -q -U git+https://github.com/huggingface/transformers.git
pip install -q -U git+https://github.com/huggingface/peft.git
pip install -q -U git+https://github.com/huggingface/accelerate.git
```

**Comment by bdqnghi** [comment]
hi, this issue will be resolved soon in the new release. The new version of transformers , peft and accelerate will be pumbed to solve the issues too.

**Comment by Paul-B98** [comment]
> hi, this issue will be resolved soon in the new release

@bdqnghi that's nice to hear. may i ask when do u plan on releasing the next release? And ist there anything that i can do to help this project?

**Comment by bdqnghi** [comment]
the issue has been resolved, please check version 1.0.2.1
